spec/services/spree_shopify_importer/data_parsers/stock_locations/base_data_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+RSpec.describe SpreeShopifyImporter::DataParsers::StockLocations::BaseData, type: :service do
+  let(:shopify_location) { build_stubbed(:shopify_location) }
+  subject { described_class.new(shopify_location) }
+  let!(:country) { create(:country, iso: 'CA', name: 'Canada') }
```

Comment:
same as above

Reply:
do you mean `let!(:country) { build_stubbed(:country, iso: 'CA', name: 'Canada') }`?

Reply:
yes, but also write something like

`expect(Spree::Country).to receive(:find_by).with(iso: iso, name: name).and_return(country)`

I'll prepare a small article about good testing examples and will share with you